skymarshal/skycmd/flags.go

```diff
@@ -242,3 +242,68 @@ func (con *Connector) newTeamConfig() (TeamConfig, error) {
 
 	return res, nil
 }
+
+type skyDisplayUserIdGenerator struct {
+	mapConnectorUserid map[string]string
+}
+
+func NewSkyDisplayUserIdGenerator(config []string) (atc.DisplayUserIdGenerator, error) {
+	generator := &skyDisplayUserIdGenerator{
+		mapConnectorUserid: map[string]string{},
+	}
+
+	for _, str := range config {
+		parts := strings.Split(str, ":")
```

Comment:
This can be avoided by using a `map[string]string` instead of `[]string` for the flag struct field; our flag and env parser already handles `foo:bar,baz:buzz` syntax for parsing into maps.

Reply:
Good to know. Fixed.